src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,21 @@
-export const PostList = () => (
-  <>Put the list here</>
+import React from 'react';
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts, users, comments }) => (
+  <div className="PostList">
+    {posts.map((post) => {
+      const user = users.find(u => u.id === post.userId);
+      const postComments = comments.filter(comment => comment.postId === post.id
```

Comment:
`prepare posts by adding a user and comments to each post (each comment has a postId);`
Do it before you put these props 3 times, in the `App` component